.github/workflows/tests.yml

```diff
@@ -13,6 +13,8 @@ jobs:
       NELNET_PAYMENT_URL: 'http://localhost:4444'
       JWT_SECRET: 'myjwtsecret'
       PATRON_ACCOUNT_BASE_URL: 'http://localhost:4567'
+      ILLIAD_API_KEY: 'YOURAPIKEY'
```

Comment:
The ILLiad key and host needed to be added. Sadly I can't just import a file with all of the keys to make testing work. :crying_cat_face: